pytorch_lightning/callbacks/quantization.py

```diff
@@ -128,8 +128,11 @@ def custom_trigger_last(trainer):
             input_compatible: preserve quant/dequant layers. This allows to feat any input as to the original model,
                 but break compatibility to torchscript
         """
-        if not isinstance(qconfig, (str, QConfig)):
-            raise MisconfigurationException(f"Unsupported qconfig: f{qconfig}.")
+        _valid_gconf_str = isinstance(qconfig, str) and qconfig in torch.backends.quantized.supported_engines
+        if not isinstance(qconfig, QConfig) and not _valid_gconf_str:
+            raise MisconfigurationException(
+                f"Unsupported qconfig: f{qconfig}.\n Try one of defaults: {torch.backends.quantized.supported_engines}"
```

Comment:
No need for the space with the line break
```suggestion
                f"Unsupported qconfig: f{qconfig}.\nTry one of defaults: {torch.backends.quantized.supported_engines}"
```

Reply:
true, but it is easier to read since it is on the same line :]